fedimint-server/src/config/api.rs

```diff
@@ -261,7 +261,17 @@ impl ConfigGenApi {
     /// Returns the consensus config hash, tweaked by our TLS cert, to be shared
     /// with other peers
     pub fn get_verify_config_hash(&self) -> ApiResult<BTreeMap<PeerId, sha256::Hash>> {
-        let state = self.require_status(ServerStatus::VerifyingConfigs)?;
+        let state = self
+            .require_status(ServerStatus::VerifyingConfigs)
```

Comment:
Might be good to have a `require_states(Vec)` in the future

Reply:
makes sense